Libs/Mesh/MeshUtils.cpp

```diff
@@ -29,4 +42,125 @@ const vtkSmartPointer<vtkMatrix4x4> MeshUtils::createIcpTransform(const vtkSmart
   return m;
 }
 
+Eigen::MatrixXd MeshUtils::distilVertexInfo(vtkSmartPointer<vtkPolyData> mesh){
```

Comment:
You can just pass `Mesh mesh` instead of `vtkSmartPointer<vtkPolyData>`. `vtkSmartPointer<vtkPolyData>` is defiend as MeshType in Mesh.h as private.

Reply:
Agreed